pkg/kubelet/prober/prober.go

```diff
@@ -93,29 +93,36 @@ func (pb *prober) probe(ctx context.Context, probeType probeType, pod *v1.Pod, s
 	}
 
 	if probeSpec == nil {
-		klog.InfoS("Probe is nil", "probeType", probeType, "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name)
+		klog.V(3).InfoS("Probe is nil", "probeType", probeType, "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name)
```

Comment:
I think we should avoid changing the existing log levels in this PR. It’s fine to add new logs, but if we stop writing logs without a good reason, it could cause issues for some users.